src/components/modals/FormalQuestions.vue

```diff
@@ -0,0 +1,1020 @@
+            >
+            <br />
+  
+            <div class="form-check form-check-radio form-check-inline">
+              <label class="form-check-label">
+                <input
+                  v-model="verificationData.customerWorkDuration"
+                  type="radio"
+                  class="form-check-input "
+                  value="Same answer with customer"
+                  name="checklist21"
+                  v-validate="'required'"
+                />
+                <span class="px-2">Same answer with customer</span>
+                <span class="form-check-sign"> </span>
+              </label>
+            </div>
+            <div class="form-check form-check-radio form-check-inline">
+              <label class="form-check-label">
+                <input
+                  v-model="verificationData.customerWorkDuration"
+                  type="radio"
+                  class="form-check-input "
+                  value="Different from what customer said"
+                  name="checklist21"
+                  v-validate="'required'"
+                />
+                <span class="px-2">Different from what customer said</span>
+                <span class="form-check-sign"> </span>
+              </label>
+            </div>
+          </div>
+        </form>
+      </div>
+      <div class="modal-footer mr-4">
+        <button
+          type="button"
+          class="btn px-5 rounded bg-danger mx-3 my-5"
+          data-dismiss="modal"
+          @click="$modal.hide('verificationForm')"
+        >
+          Close
+        </button>
+        <button type="button" class="btn px-4 bg-default my-5" @click="save">
+          Save changes
+        </button>
+      </div>
+    </div>
+  </transition>
+    
+  </template>
+  
+  <script>
+  import { get, post } from "../../utilities/api"
+  import { EventBus } from '../../utilities/event-bus'
+  import Vue from 'vue'
+  import InformalQuestions from './InformalQuestions.vue';
+  import { mapGetters } from 'vuex'
+  
+  export default {
+    props: {
+    customer: {
+      required: true,
+      default: {},
+    },
+  },
+    data() {
+      return {
+        verifiedOptions: ["Home", "Business", "Office", "Other"],
+        vAgents: null,
+        verificationData: {},
+        locationDurationOptions: ["less than 6 months","6months - 1 year","More than 1 year",],
+        customerWorkingYearsOptions: ["less than 6 months","6months - 1 year","More than 1 year",],
+        showCustomer: false,
+        status: "Formal"
+      }
+    },
+    components: {
+    InformalQuestions,
+  },
+  computed: {
+    ...mapGetters(["auth"]),
+    full() {
+      return this.$route.meta.mode === "full"
+    },
+  },
+
+  created() {
+    $(".tooltip").remove()
+    if (this.customer) this.setCustomer(this.customer)
+    EventBus.$on("customer", customer => this.setCustomer(customer))
+    this.addCustomerOptionsModalsToDom()
```

Comment:
Remove this. It has no use. It is causing console error

Reply:
okay